src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/LocationOptionMeta.cs

```diff
@@ -1,12 +1,190 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
-public class LocationOptionMeta
+public abstract class LocationOptionMeta
 {
-    public required string PublicId { get; set; }
-
-    public required int PrivateId { get; set; }
+    public int Id { get; set; }
```

Comment:
`Guid`

Reply:
**Discussed:**

Only using `Guids` for publicly available IDs. Using `int` for strictly internally available IDs as it is more performant.